Eplant/views/eFP/Viewer/EFPViewerCitation.tsx

```diff
@@ -0,0 +1,57 @@
+import GeneticElement from '@eplant/GeneticElement'
+import { getCitation } from '@eplant/util/citations'
+import React from 'react'
```

Comment:
I get this error `React' must be in scope when using JSX` if I don't have the import React line (not sure why might be an ESLint thing because I don't think we should need to do this with our React version)

Reply:
Yeah, after React 17 we [apparently don't need this flag anymore](https://stackoverflow.com/questions/42640636/react-must-be-in-scope-when-using-jsx-react-react-in-jsx-scope). In your eslint file add the rule `rules { "react/react-in-jsx-scope": "off",}`

Reply:
and add a lil `  "extends": ["plugin:react/jsx-runtime"]` if need be.